public/js/gogs.js

```diff
@@ -830,6 +830,10 @@ $(document).ready(function () {
 
     // Highlight JS
     if (typeof hljs != 'undefined') {
+        // creating aliases for languages
```

Comment:
I didn't know that it was already implemented. ¯\_(ツ)_/¯ Updated to use that package, much better.

I also did this patch to Highlight.js, so futures version will have these alises by default: https://github.com/isagalaev/highlight.js/pull/1069/files

There was just a little gotcha: I had to move `highlight.go` to its own package, because we got a cycle import, which isn't allowed in Go, on importing `template` in `models` since `models` also import `template`.